README.md

```diff
@@ -58,6 +65,11 @@ Alternatively, you can use the go install method to compile the project using yo
 
     go install github.com/vektra/mockery/v2@latest
 
+Requirement
+-----------
+
+For mockery to correctly generate mocks, the command has to be run on a module (i.e. your project has to have a go.mod file)
```

Comment:
Actually, maybe this should belong as a note under the `Examples` section. But good call, people do get caught by this 👍🏻